CMakeLists.txt

```diff
@@ -369,3 +369,19 @@ install(
   RUNTIME DESTINATION bin
   BUNDLE DESTINATION bin
   LIBRARY DESTINATION lib)
+
+# for creating DEB and RPM packages
+set(CPACK_PACKAGE_CONTACT "ruby.mamistvalove@gmail.com")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY
+    "Notes is a simple and beautiful note taking app.")
+set(CPACK_PACKAGE_DESCRIPTION
+    "Notes is a simple and beautiful note taking app. It is cross-platform and written in C++ using the Qt framework."
+)
+# set dependencies based on the Qt version
+if(${QT_VERSION_MAJOR} EQUAL 6)
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "libqt6widgets6 (>= 6.0), libqt6gui6 (>= 6.0), libqt6network6 (>= 6.0), libqt6sql6 (>= 6.0), libqt6core6 (>= 6.0), sqlite3, libqt6sql6-sqlite (>= 6.0)")
```

Comment:
Perhaps instead of installing the whole `sqlite3` package (which include some binaries we don't need, like `sqlite3` and `sqldiff`), we could just use `libsqlite3-0` instead?

Reply:
I noticed we might not actually need `sqlite3`, as it isn't even included in the list provided by `ldd`. I think libqt6sql6 contains everything we need. Not sure if this is the same case for Qt5.

Reply:
Indeed. For both `libqt5sql5` and `libqt6sql6`, `libsqlite3-0` is already listed as a dependency.